src/frontend/src/lib/components/tokens/TokensMenu.svelte

```diff
@@ -1,34 +1,42 @@
 <script lang="ts">
-	import { IconDots, Popover } from '@dfinity/gix-components';
+	import { Popover } from '@dfinity/gix-components';
 	import { erc20UserTokensNotInitialized } from '$eth/derived/erc20.derived';
+	import IconManage from '$lib/components/icons/IconManage.svelte';
 	import ManageTokensMenuButton from '$lib/components/manage/ManageTokensMenuButton.svelte';
 	import TokensZeroBalance from '$lib/components/tokens/TokensZeroBalance.svelte';
+	import TokensZeroBalanceCheckbox from '$lib/components/tokens/TokensZeroBalanceCheckbox.svelte';
+	import ButtonIcon from '$lib/components/ui/ButtonIcon.svelte';
 	import Hr from '$lib/components/ui/Hr.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 
 	let visible = false;
 	let button: HTMLButtonElement | undefined;
 </script>
 
-<button
-	class="icon"
-	bind:this={button}
-	on:click={() => (visible = true)}
-	aria-label={$i18n.navigation.alt.menu}
-	disabled={$erc20UserTokensNotInitialized}
-	class:opacity-10={$erc20UserTokensNotInitialized}
->
-	<IconDots size="24" />
-</button>
+<div class="hidden sm:block">
+	<TokensZeroBalanceCheckbox />
+</div>
 
-<Popover bind:visible anchor={button} invisibleBackdrop direction="rtl">
-	<div class="flex flex-col gap-3">
-		<TokensZeroBalance />
+<div class="block sm:hidden">
+	<ButtonIcon
+		bind:button
+		on:click={() => (visible = true)}
+		ariaLabel={$i18n.navigation.alt.menu}
+		disabled={$erc20UserTokensNotInitialized}
+		link={false}
+	>
+		<IconManage slot="icon" />
+	</ButtonIcon>
 
-		<div class="my">
-			<Hr />
-		</div>
+	<Popover bind:visible anchor={button} invisibleBackdrop direction="rtl">
+		<div class="flex flex-col gap-3">
+			<TokensZeroBalance />
+
+			<div class="my">
```

Comment:
`my` exists? it's a shorthand for `my-0`?

Reply:
that was actually there before, but you see modified because I wrapped the entire element and it shifted (Github strangly shows it as modified).

However, it is not a class, so I will remove it, since we are here